src/components/portfolio/PortfolioCard.tsx

```diff
@@ -0,0 +1,350 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+import { roundPercentage } from '../../util/Numbers';
+import PositiveChangeIcon from '../../assets/svg/positive_change_chevron.svg';
+import NegativeChangeIcon from '../../assets/svg/negative_change_chevron.svg';
+import { TokenData } from '../../data/TokenData';
+import { FeeTier, PrintFeeTier } from '../../data/BlendPoolMarkers';
+import { SiloData } from '../../data/SiloData';
+import { prominent } from 'color.js';
+
+export const RESPONSIVE_BREAKPOINT_MEDIUM = '1200px';
+export const RESPONSIVE_BREAKPOINT_SMALL = '768px';
```

Comment:
Should these constants stay exclusive to the Portfolio card or should they apply for other web app components? I.e. is this file the right place for the constants?

Reply:
I am not sure if they will apply to other web app components. I say we keep them here for now and move them if other components (outside of the portfolio components). What are your thoughts on doing this? @adamegyed

Reply:
Added.